tree/treeplayer/inc/ROOT/TCsvDS.hxx

```diff
@@ -15,7 +16,7 @@ namespace TDF {
 class TCsvDS final : public ROOT::Experimental::TDF::TDataSource {
 
 private:
-   typedef std::vector<void *> Record;
+   using Record = std::vector<void *>;
```

Comment:
Curiosity: what is the advantage of one vs the other?

Reply:
In TDF we decided to adopt only aliases rather than typedefs. There are a few advantages of using aliases, e.g. enhanced clarity in some cases. In this particular case, it's just consistency with the rest of the code, no real advantage :-)